content/docs/3.0/contribute/get-started.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Contribute
+description: How to be a part of TiKV
+menu:
+    nav:
+        parent: Docs
+        weight: 6
+    docs:
+        weight: 6
+---
+
+As an open source project, TiKV cannot grow without support and participating of contributors from the community. If you would like to contribute to the TiKV code, our documentation, or even the website, we would appreciate your help. And we are glad to provide any support along the way.
+
+## How to be a TiKV Contributor
+
+If a PR (Pull Request) submitted to the [TIKV](https://github.com/tikv) projects by you is approved and merged, then you become a TiKV Contributor.
+
+## Pick an area to contribute
+
+You can choose the one of the following areas to contribute:
+
+- TiKV core repositories
```

Comment:
Added Libraries, Clients, Meetups/Events, RFCs, but not sure about Query Layers since we already have that info in Reference and it is not part of TiKV. Also, I need your help updating the Art part.